observations/unhealthy-scale-out-mid-deploy.md

```diff
@@ -0,0 +1,67 @@
+# What happens when a scale-out event occurs mid-deployment of an unhealthy artifact?
+
+In this test we deployed [application version](../dist) `ABC` to `500` in the stack
+[`ScalingAsgRollingUpdate`](../packages/cdk/lib/scaling-asg-rolling-update.ts) (CFN stack `playground-CODE-scaling-asg-rolling-update`).
+
+The aim of this test is to understand the behaviour in the ["Ophan scenario"](https://github.com/guardian/riff-raff/issues/1342).
+
+## Highlights
+The deployment leaves the service over-capacity.
```

Comment:
I guess this is similar to the findings in https://github.com/guardian/testing-asg-rolling-update/blob/main/observations/healthy-to-unhealthy.md#highlights (but at least in this case we know it will eventually scale down automatically!)